src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1363,6 +1363,7 @@ def db_update(
     if _should_show_backup_storage_redundancy_warnings(location):
         if storage_account_type == 'GRS':
             _backup_storage_redundancy_specify_geo_warning()
+    print(storage_account_type)
```

Comment:
Is this for debug?

Reply:
yeah my bad I'm taking it out ... no need to show customer 👍